frontend/src/app/[locale]/search/page.tsx

```diff
@@ -1,6 +1,6 @@
 import { Metadata } from "next";
 import QueryProvider from "src/app/[locale]/search/QueryProvider";
-import withFeatureFlag from "src/hoc/search/withFeatureFlag";
+import withFeatureFlag from "src/hoc/withFeatureFlag";
```

Comment:
not sure how these changes didn't get through - I may have made a mistake in a rebase somewhere along the way